boot/environ/lib/io/display/display.c

```diff
@@ -114,28 +115,35 @@ DsppLoadFontFile (
     DirectoryLength = wcslen(FontDirectory);
 
     /* Safely add them up*/
-    Status = RtlULongAdd(NameLength, DirectoryLength, &TotalLength);
+    Status = RtlSIZETAdd(NameLength, DirectoryLength, &TotalLength);
```

Comment:
Do you mean RtlSizeTAdd?

Reply:
Nope. They both exist, but SizeT uses size_t* and SIZET uses SIZE_T* as 3rd parameter and for whatever reasons MS has decided to declare SIZE_T different from size_t, so that their pointers are incompatible.

Reply:
Where is "RtlSIZETAdd" defined?

Reply:
I fixed the remaining ones. MSDN is wrong. Please use git.reactos.org, or the search tool of your choice and the Windows SDK headers.

Reply:
Indeed that's correct that MSDN is not accurate here.
To my discharge, git.reactos.org shows nothing for RtlSizeTAdd (or any version of the casing) here: https://git.reactos.org/?p=reactos.git&a=search&h=HEAD&st=grep&s=rtlsizetadd&sr=1
Looks like the implementation of these helpers (in ntintsafe.h) does not exist yet in trunk.